scripts/run_integration_tests.sh

```diff
@@ -24,6 +24,12 @@ script_utc_start_time=$(date -u +"%Y%m%dT%H%M%S")
 
 mismatch_found=false
 
+if [ -z "$AWS_SECRET_ACCESS_KEY" ]; then
+    echo "No AWS credentials were found in the environment."
+    echo "Note that only Datadog employees can run these integration tests."
```

Comment:
It's possible for a non-Datadogger to run these tests if they supply their own AWS credentials, no?

Reply:
Not without modification. The tests expect a specific S3 bucket and IAM role to be in place in the AWS account.

Reply:
Ah gotcha. Makes sense!